utils/docker/dev_peer_cc-builder/Dockerfile

```diff
@@ -191,4 +191,7 @@ RUN apt-get install -y -q \
 	docker-compose\
 	${APT_DEV_ADD_PKGS}
 
+RUN GO111MODULE=on \
```

Comment:
Do we need it from source (and latest)?  On my host it was convenient (and providing a well-defined version) to do a `snap install yq`.  Mainly asking whethere we _must_ have the go getted binary in our PATH. I guess if we don't fear version incompatibility then it probably shouldn't matter which version we take; if do fear version incompatibility, then i guess we should also ask here for a specfici version.

(Asking also as package.sh gives me problems with `jq` (yes, i know it is `jq`, not `yq`) and was wondering whether that problem might maybe be related to versions of jq as presumably the script does work for you ...)

Reply:
Oh I tried to install snap in dev docker but for some reason it just did not work, so I stoped bothering with it and tried via go get. 

BTW on Mac:
```
yq --version
yq version 3.4.0
```

Reply:
via snap on 18.04 i get `3.4.1`  which is the same i get when i do right now a go get install. So at least right now all three version differ only by patch-level which probably should be fine (contrary to jq where we differ in major version)